packages/target-decisioning-engine/src/geoProvider.js

```diff
@@ -126,9 +126,13 @@ export function GeoProvider(config, artifact) {
         headers[HTTP_HEADER_FORWARDED_FOR] = geoRequestContext.ipAddress;
       }
 
-      return fetchApi(geoLookupPath, {
-        headers
-      })
+      const options = { headers };
+
+      if (config.proxyAgent != null) {
```

Comment:
I'd suggest simplifying a bit
```suggestion
      if (config.proxyAgent) {
```

Reply:
for consistency let's use `isDefined` please.

```js
if(isDefined(config.proxyAgent)) {
```